src/Umbraco.Web.UI.Client/src/views/components/property/property-actions/umb-property-actions.html

```diff
@@ -1,11 +1,23 @@
 <div class="umb-property-actions" ng-if="vm.actions.length > 0">
-    <button type="button" class="btn-reset umb-outline umb-property-actions__toggle" ng-click="vm.toggle()" localize="title" title="propertyActions_tooltipForPropertyActionsMenu"><i></i><i></i><i></i></button>
+    <umb-button-ellipsis
+        css-class="umb-outline umb-property-actions__toggle"
+        action="vm.toggle()"
+        mode="small"
+        text="Open Property Actions"
+        label-key="propertyActions_tooltipForPropertyActionsMenu"
+        >
+    </umb-button-ellipsis>
 
     <div class="umb-property-actions__menu" role="menu" ng-if="vm.isOpen" on-outside-click="vm.close()" on-close="vm.close()" deep-blur="vm.close()">
-        <button
-            class="umb-property-actions__menu-open-toggle"
-            ng-click="vm.close()" tabindex="-1"><i></i><i></i><i></i>
-        </button>
+
+        <umb-button-ellipsis
+            css-class="umb-button-ellipsis--absolute umb-property-actions__menu-open-toggle"
+            action="vm.close()"
+            mode="small"
+            text="Close Property Actions"
```

Comment:
@BatJan Aren't you missing the `label-key` here? 😁

Reply:
Doh! Yes I am :-)

Reply:
@nul800sebastiaan  Should be fixed now 👍